webapp/README.md

```diff
@@ -115,3 +115,9 @@ This plugin registers a post dropdown action that, when clicked, triggers the ro
 This plugin registers a file upload action that, when clicked, triggers the root component:
 
 ![file upload](docs/file_upload.png)
+
+## Link Tooltip
+
+This plugin registers a link tooltip component, whose content is completely customizable and that is displayed when the mouse pointer is moved onto a link in a post.
```

Comment:
By 'content', I assume this is the icon, colour and the text? If so, propose a minor update with

```
This plugin registers a link tooltip component, whose icon, text and colour are customizable. It is displayed when the mouse cursor hovers over a link in a post.
```

Reply:
Ah okay. Then just minor proposed adjustment below and looks good to me :)

```
This plugin registers a link tooltip component, whose content is fully customizable. It is displayed when the mouse cursor hovers over a link in a post.
```

Reply:
Done ;)